crates/proof-of-sql/Cargo.toml

```diff
@@ -87,6 +87,11 @@ name = "criterion_benches"
 harness = false
 required-features = [ "blitzar" ]
 
+[[bench]]
```

Comment:
We need to rename the other criterion bench then since `bench_append_rows` is also criterion.